Taxi/Taxi/Presenter/SignUpCodeView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SignUpCodeView.swift
+//  Taxi
+//
+//  Created by sanghyo on 2022/06/14.
+//
+
+import SwiftUI
+
+struct SignUpCodeView: View {
+    @Environment(\.dismiss) private var dismiss
+    @State private var signUpCode: String = ""
+    @State private var codeState: FieldState = .normal
+    @State private var isActive: Bool = false
+    private let developerCode: String = "popopot"
```

Comment:
현재 signUpCode 변수가 사용자가 텍스트필드에 입력한 값을 저장하는 변수로 사용되고 있고, developerCode가 가입코드로 이용되고 있는데, 변수명이 적절하지 않아 보입니다!
변수명을 보고는 역할을 예상하기가 힘들어요.

```suggestion
    @State private var codeInput: String = ""
    @State private var codeState: FieldState = .normal
    @State private var isActive: Bool = false
    private let signUpCode: String = "popopot"
```
이정도로 바꾸면 signUpCode가 가입 코드라는 의미를 명확하게 전달하고, codeInput에 입력값을 받는구나 하고 알 수 있을 것 같네요!

Reply:
변수명에 대한 생각을 좀 더 많이 해보고 이름을 지어야겠습니다. 앞으로도 적절한 명명법 고민 충분히 해보겠습니다 ~